Cargo.toml

```diff
@@ -59,7 +59,7 @@ tower = { version = "0.4.12", features = ["full"] }
 tracing = "0.1.34"
 tracing-subscriber = { version = "0.3.16" , features = ["registry", "env-filter"]}
 dyn-clone = "1.0.9"
-realm_io = "0.3.5"
+realm_io = { git = "https://github.com/howardjohn/realm", branch = "io/copy-bytes-sent" }
```

Comment:
Will you file a pr to realm?

Reply:
https://github.com/zhboner/realm/pull/95. We can see what they say before we merge this, hopefully they are ok with it

Reply:
Merged, switched to real dep